packages/taquito-michelson-encoder/test/tokens/timestamp.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { TimestampToken } from '../../src/tokens/comparable/timestamp';
+
+describe('Timestamp token', () => {
+  let token: TimestampToken;
+
+  beforeEach(() => {
+    token = new TimestampToken({ prim: 'timestamp', args: [{ string: '1638554817'}], annots: [] }, 0, null as any);
+  });
+
+  describe('Execute', () => {
+    it('Should return the RFC3339 time format when type is int and value is UNIX timestamp', () => {
+      const result = token.Execute({ int: 1638555790 } as any);
```

Comment:
{ int: 1638555790 } should be { int: '1638555790' } and thus no need to cast to any.